yb-voyager/cmd/live_migration.go

```diff
@@ -54,7 +55,12 @@ func streamChanges(state *ImportDataState, tableNames []string) error {
 	log.Infof("NUM_EVENT_CHANNELS: %d, EVENT_CHANNEL_SIZE: %d, MAX_EVENTS_PER_BATCH: %d, MAX_INTERVAL_BETWEEN_BATCHES: %d",
 		NUM_EVENT_CHANNELS, EVENT_CHANNEL_SIZE, MAX_EVENTS_PER_BATCH, MAX_INTERVAL_BETWEEN_BATCHES)
 	tdb.PrepareForStreaming()
-	err := state.InitLiveMigrationState(migrationUUID, NUM_EVENT_CHANNELS, bool(startClean), tableNames)
+
+	renamedTables := make([]string, 0)
+	for _, tableName := range tableNames {
+		renamedTables = append(renamedTables, renameTableIfRequired(tableName))
+	}
+	err := state.InitLiveMigrationState(migrationUUID, NUM_EVENT_CHANNELS, bool(startClean), renamedTables)
```

Comment:
Why are we renaming here? Because essentially we're passing renameconfig to dbzm, so we will always get root tables? Pls add a comment here to explain..
Also, should we do a unique before passing? I assume renamedTables might have duplicates
?